ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/ldes/retentionpolicy/versionbased/ReverseTimeStampComparator.java

```diff
@@ -0,0 +1,12 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.domain.ldes.retentionpolicy.versionbased;
+
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.tree.member.entities.Member;
+
+import java.util.Comparator;
+
+public class ReverseTimeStampComparator implements Comparator<Member> {
```

Comment:
I'm a bit in dubio whether this should just be a "TimeStampComparator" and that you add a reverse() on call, or this approach. But since it works, i'm ok with this